src/content/reference/react/useCallback.md

```diff
@@ -101,9 +101,9 @@ function ProductPage({ productId, referrer, theme }) {
   );
 ```
 
-You've noticed that toggling the `theme` prop freezes the app for a moment, but if you remove `<ShippingForm />` from your JSX, it feels fast. This tells you that it's worth trying to optimize the `ShippingForm` component.
+Anda akan menyadari bahwa mengganti nilai prop `theme` membuat aplikasi terasa berhenti sejenak, tapi jika Anda hapus `<ShippingForm />` dari JSX Anda, itu akan terasa lebih cepat. ini memberitahu Anda bahwa itu layak dicoba untuk optimasi komponen `ShippingForm`.
 
-**By default, when a component re-renders, React re-renders all of its children recursively.** This is why, when `ProductPage` re-renders with a different `theme`, the `ShippingForm` component *also* re-renders. This is fine for components that don't require much calculation to re-render. But if you verified a re-render is slow, you can tell `ShippingForm` to skip re-rendering when its props are the same as on last render by wrapping it in [`memo`:](/reference/react/memo)
+**Secara default, ketika sebuah komponen melakukan render ulang, React akan melakukan render ulang pada seluruh komponen turunannya secara rekursif** inilah kenapa, ketika `ProductPage` render ulang dengan `theme` berbeda, komponen `ShippingForm` *juga* render ulang. Ini bagus untuk komponen yang tidak memerlukan banyak perhitungan untuk render ulang. tapi jika anda telah memverifikasi bahwa render ulang itu lambat, Anda dapat memberitahu `ShippingForm` untuk melewati render ulang ketika props itu sama seperti render sebelumnya dengan cara bungkus itu di dalam [`memo`:](/reference/react/memo)
```

Comment:
```suggestion
**Secara *default*, ketika sebuah komponen melakukan *render* ulang, React akan melakukan *render* ulang pada seluruh komponen turunannya secara rekursif** inilah kenapa, ketika `ProductPage` *render* ulang dengan `theme` berbeda, komponen `ShippingForm` juga *render* ulang. Ini bagus untuk komponen yang tidak memerlukan banyak perhitungan untuk *render* ulang. tapi jika anda telah memverifikasi bahwa *render* ulang itu lambat, Anda dapat memberitahu `ShippingForm` untuk melewati *render* ulang ketika *props* itu sama seperti *render* sebelumnya dengan cara bungkus itu di dalam [`memo`:](/reference/react/memo)
```

Reply:
i just wanna ask something. in line 106. i translate from *also* to *juga* but from your comment become "juga". but in line 116, i translate from *same* to *sama* not become "sama" but still *sama*. so, i just wanna make sure, which is true sentences? in line 106 or 116?

Reply:
sorry mas @r17x maybe you passed what i asked in this comment.

Reply:
I think we can remove `"` when translated yet to Bahasa, What do you think? @intanirnanda5796

Reply:
hmmm ok will do it @r17x . thanks mas.